apps/api/src/app/events/e2e/delay-events.e2e.ts

```diff
@@ -12,8 +12,10 @@ import {
 import { UserSession, SubscribersService } from '@novu/testing';
 import { StepTypeEnum, DelayTypeEnum, DigestUnitEnum, DigestTypeEnum } from '@novu/shared';
 import { QueueService } from '@novu/application-generic';
+import { setTimeout } from 'timers/promises';
 
 const axiosInstance = axios.create();
+const promiseResolveTimeout = (ms: number) => setTimeout(ms);
```

Comment:
This is not needed. Just call `await setTimeout` directly wherever it is needed.

Reply:
Done @p-fernandez